src/decide.js

```diff
@@ -52,10 +47,6 @@ async function assessResultHelper(result, ogInfo, hashesToExclude) {
 	// if you got rate limited or some other failure
 	if (!info) return info;
 
-	if (info.length !== ogInfo.length) {
-		logReason("it has a different size");
-		return null;
-	}
```

Comment:
Could you explain why we don't need this part anymore? One thing I want to watch out for here is re-subtitled files where they took the file and added subtitles for a certain language (say, Romanian). The info.length property also only applies to single-file torrents so I'm not sure that this would affect your target in this PR.

Reply:
It's checking the whole torrent content size, so if the new torrent has less files, it would never match.

Reply:
Better check the size per-file in the compareFileTrees. This one is redundant I think. It's checking the whole torrent content size, so if the new torrent has less files, it would never match. Goes against this PR.

Reply:
Here's the relevant portion of the BT spec:

> There is also a key length or a key files, but not both or neither. If length is present then the download represents a single file, otherwise it represents a set of files which go in a directory structure.

> In the single file case, length maps to the length of the file in bytes.

> For the purposes of the other keys, the multi-file case is treated as only having a single file by concatenating the files in the order they appear in the files list. The files list is the value files maps to, and is a list of dictionaries containing the following keys:

> length - The length of the file, in bytes.

> path - A list of UTF-8 encoded strings corresponding to subdirectory names, the last of which is the actual file name (a zero length list is an error case).

> In the single file case, the name key is the name of a file, in the muliple file case, it's the name of a directory.

(From http://www.bittorrent.org/beps/bep_0003.html)

length will only be set on single-file torrents so we will still want to check it, I think.

Reply:
I checked that on multi-file torrent, it had length set to sum of size of every file. I think length from parse-torrent is not the same as length from the protocol. It's just size of the torrent.

Reply:
Ah yes, you're right. https://github.com/webtorrent/parse-torrent/blob/master/index.js#L182-L193

Sounds good then!